README.md

```diff
@@ -50,4 +50,4 @@ Once you’ve configured and committed your workflow to the default branch of yo
 
 ## Implementation Details
 
-Internally, the Stark Github action runs its code inside a `node-16` process. The process passes everything written within the `setup`, `prebuild`, `build`, `serve`, and `cleanup` parameters to the default shell on the image. That gives you the ability to run any kind of script you would like within these commands. The `serve` argument is slightly different: it starts a detached process that is killed automatically when our step ends. If you would like to explicitly kill the process (for instance, to safely close some database connections), use the `cleanup` script.
+Internally, the Stark Github action runs its code inside a `node-18` process. The process passes everything written within the `setup`, `prebuild`, `build`, `serve`, and `cleanup` parameters to the default shell on the image. That gives you the ability to run any kind of script you would like within these commands. The `serve` argument is slightly different: it starts a detached process that is killed automatically when our step ends. If you would like to explicitly kill the process (for instance, to safely close some database connections), use the `cleanup` script.
```

Comment:
I wonder if we should omit the version entirely so it doesn't need to be updated every time.

Reply:
I could remove this but I feel it is a vital piece of information. In fact that gatsby project made me realise that I hadn't carefully factored in node versioning, but now the idea will be to have a recent LTS base image.
So, 20.x will go into LTS from this Oct and will stay there for another year at least before moving to maintenance. By that, our next potential update to this will be Oct and then 2 years later for 22.x. (Unless everyone moves to Deno, I would want to see that).